docs/topics/installing-mta-disconnected-environment.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="installing-mta-disconnected-environment_{context}"]
+= Installing {ProductShortName} on a disconnected environment
+
+On a connected device, first download and save the {ProductShortName} binary. Then download and save the Podman images, the {ProductShortName} Podman image and the provider image that you need.
+
+* Download the required {ProductShortName} CLI binary from the link:https://developers.redhat.com/products/mta/download[{ProductName} Red Hat Developer page]:
+** CLI for Linux x86_64 
+** CLI for Linux aarch64
+** CLI for MacOS x86_64
+** CLI for MacOS aarch64
+** CLI for Windows x86_64
+** CLI for Windows aarch64
+
+* On a connected device, download and save the images.
+
+* Copy the binary to the disconnected device.
+
+* In addition, you must save and import the associated container images by using Podman.
+
+== Downloading the Podman images
+
+.Prerequisites
+
+* Podman installed. For more information, see link:https://podman.io/[Podman].
+
+.Procedure
+
+. Use Podman to authenticate to link:registry.redhat.io[registry.redhat.io]:
++
+[source,terminal]
+----
+$ podman login registry.redhat.io
+----
+
+. Enter your username and then your password for registry.redhat.io:
++
+[source,terminal]
+----
+Username: <registry_service_account_username>
+Password: <registry_service_account_password>
+----
++
+You should see the following output:
++
+[source,terminal]
+----
+Login Succeeded!
+----
+
+. Use Podman to pull the image from the registry:
++
+[source,terminal]
+----
+$ podman pull registry.redhat.io/mta/mta-rhel9-operator:7.1.0
```

Comment:
@ibragins , is operator correct here or should this be mta-cli image?

Reply:
@anarnold97 , from Dylan's [comment](https://issues.redhat.com/browse/MTA-3245?focusedId=25265800&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-25265800) on MTA-3245, this should be mta-cli.

```suggestion
$ podman pull registry.redhat.io/mta/mta-cli-rhel9:7.1.0
```